core/oracle/token_price_manager_test.go

```diff
@@ -29,49 +39,91 @@ func TestTokenManager(t *testing.T) {
 			},
 		}
 
+		sushiswap := &sushiswap.MockSushiSwapManager{
+			GetPriceFromSushiswapFunc: func(tokenAddress string) (*types.TokenPrice, error) {
+				price, ok := new(big.Int).SetString("1", 10)
+				require.Equal(t, ok, true)
+				return &types.TokenPrice{
+					Price: price,
+					Id:    "ETH",
+				}, nil
+			},
+		}
+
+		uniswap := &uniswap.MockNewUniwapManager{
+			GetPriceFromUniswapFunc: func(tokenAddress string) (*types.TokenPrice, error) {
+				price, ok := new(big.Int).SetString("1", 10)
+				require.Equal(t, ok, true)
+				return &types.TokenPrice{
+					Price: price,
+					Id:    "ETH",
+				}, nil
+			},
+		}
+
 		dbInstance := database.NewDb(&cfg)
 		err := dbInstance.Init()
 		if err != nil {
 			panic(err)
 		}
 
-		priceManager := NewTokenPriceManager(cfg, dbInstance, net)
+		priceManager := NewTokenPriceManager(cfg, dbInstance, net, uniswap, sushiswap)
 		price, err := priceManager.GetTokenPrice("ETH")
 		require.Nil(t, err)
-		require.Equal(t, "2410875945408672038912", price.String())
-
-		price, err = priceManager.GetTokenPrice("BTC")
-		require.Nil(t, err)
-		require.Equal(t, "36367076791144564129792", price.String())
+		require.Equal(t, "1", price.String())
```

Comment:
Don't change big value -> small value. The previous test uses big number because it's close to actual value.